Gemfile

```diff
@@ -32,7 +33,7 @@ group :development do
   gem 'better_errors'
   gem 'binding_of_caller'
   gem 'quiet_assets', '~> 1.1.0'
-  gem 'govuk-lint', '0.5.0'
+  gem 'govuk-lint'
```

Comment:
Yup, I understand where you are coming from on this.

The way I have been looking at this so far is: `Gemfile.lock` already remembers versions, so I have reserved pinned versions in the Gemfile for 2 reasons:

- for gems that we have a hard dependency on (e.g. if I bump my version of gem X nothing will work and we can't commit the time to fix that right now);
- rails (pinned to minor version only so we get patch version upgrades without touching the Gemfile).

This way, we can run `bundle update my-awesome-gem` without having to edit the Gemfile (or find out what the latest version is). If all goes well, we only commit the `Gemfile.lock`.

What do you think?